<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-08-25T14:51:54 IST" name="Default suite" finished-at="2020-08-25T14:52:02 IST" duration-ms="8326">
    <groups>
    </groups>
    <test started-at="2020-08-25T14:51:54 IST" name="Default test" finished-at="2020-08-25T14:52:02 IST" duration-ms="8326">
      <class name="Academy.ValidateTitle">
        <test-method is-config="true" signature="browserInitialise()[pri:0, instance:Academy.ValidateTitle@6f3c660a]" started-at="2020-08-25T14:51:54 IST" name="browserInitialise" finished-at="2020-08-25T14:52:02 IST" duration-ms="8080" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserInitialise -->
        <test-method signature="getHeader()[pri:0, instance:Academy.ValidateTitle@6f3c660a]" started-at="2020-08-25T14:52:02 IST" name="getHeader" finished-at="2020-08-25T14:52:02 IST" duration-ms="61" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getHeader -->
        <test-method signature="getTitle()[pri:0, instance:Academy.ValidateTitle@6f3c660a]" started-at="2020-08-25T14:52:02 IST" name="getTitle" finished-at="2020-08-25T14:52:02 IST" duration-ms="49" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitle -->
        <test-method signature="getTitleFailed()[pri:0, instance:Academy.ValidateTitle@6f3c660a]" started-at="2020-08-25T14:52:02 IST" name="getTitleFailed" finished-at="2020-08-25T14:52:02 IST" duration-ms="23" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATUREDCOURSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATUREDCOURSES] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at Academy.ValidateTitle.getTitleFailed(ValidateTitle.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTitleFailed -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Academy.ValidateTitle@6f3c660a]" started-at="2020-08-25T14:52:02 IST" name="tearDown" finished-at="2020-08-25T14:52:02 IST" duration-ms="78" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Academy.ValidateTitle -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
